### ISSUE TITLE
Search field position unstable when using splits

### ISSUE THREAD

**Issue by bfulop** [veonim/veonim#425]
If I split the Veonim window into multiple views (e.g. two on top, one on the bottom) and start searching (with `/`) the search field will start to jump around between the three splits.

UPDATE: The search input wants to go down to the bottom split. If I start a search in the bottom split, then the search input field stays at the correct position, but if I invoke it on a top split, then it will want to position itself on the bottom split. 

(aside note: I've been using Veonim full time this week and it's great, and totally usable. I understand that it's in alpha stage. Just let me know if it's too early to report these small bugs)

**Comment by breja** [comment]
This should be fixed now on master. Let me know how it behaves now.

Sorry, my bad - this was an experimental feature not quite ready for the light of day just yet. Not your fault though - it was not obvious in any way. No worries - we'll make this feature prod-ready now. 😄 

I have the habit of keeping wip/experimental features in master. Gonna keep my crazy experiments out of master or at least behind feature/experimental flags from now on.

I'm happy to hear that Veonim is working for you! It's motivational to see others using the app (or trying to, hah). I don't mind the feedback at all. I just feel bad that the app is not quite as polished as I'd like it to be. But hopefully it gets better little-by-little.

**Comment by bfulop** [comment]
Thanks for fixing this so quickly! Working perfect here too!

Veonim is already great and it's just completely mind blowing how far you got already in such a short time! I'm an Onivim user (and continuing Patreon supporter) but wanted to try out Veonim after following the repo for a bit. I felt so comfortable with it that I just kept it using it. I got spoiled by the UI, the snappiness, multiple workspaces, workspace search.  😋

Will continue to report then if I see some issues
